src/Tasks/ResolveAssemblyReferences/Contract/IResolveAssemblyReferenceTaskHandler.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.Tasks.ResolveAssemblyReferences.Contract
+{
+    public interface IResolveAssemblyReferenceTaskHandler : IDisposable
+    {
+    }
```

Comment:
I'm assuming you're planning to fill this in in a later PR, right?

Reply:
Yes